src/lib_c.cr

```diff
@@ -1,5 +1,5 @@
 {% if flag?(:win32) %}
-  @[Link("libcmt")]
+  @[Link("msvcrt")]
```

Comment:
I think there should be a `-Dwin_MT` to link `libcmt` for those that want to use `/MT` builds.

Reply:
Probably. But not sure I like that exact name and approach to choosing this option.

But it *is* true that this is different from `--static`.
(Wait, does that one even apply to Windows? It's *.lib either way? And I think we've been linking statically without `--static`)